paper.bib

```diff
@@ -75,6 +75,82 @@ @ARTICLE{fhd
   pages = {17},
   doi = {10.1088/0004-637X/759/1/17},
   adsurl = {http://adsabs.harvard.edu/abs/2012ApJ...759...17S},
-  adsnote = {Provided by the SAO/NASA Astrophysics Data System}
+  adsnote = {Provided by the SAO/NASA Astrophysics Data System},
   url = {https://github.com/EoRImaging/FHD}
 }
+
+@TECHREPORT{ms,
+  author = {{Kemball}, A.~J. and {Wieringa}, M.~H.},
+  title = "{MeasurementSet definition version 2.0}",
+  year = 2000,
+  month = jan,
+  file = {https://casa.nrao.edu/Memos/229.ps.gz},
+  url = {https://casper.berkeley.edu/astrobaki/index.php/AIPY}}
+}
+
+@TECHREPORT{mir,
+  author = {{Qi}, C.},
+  title = "{The MIR Cookbook}",
+  year = 2022,
+  month = mar,
+  file = {https://lweb.cfa.harvard.edu/~cqi/mircook.pdf},
+  url = {https://lweb.cfa.harvard.edu/~cqi/mircook.html}}
+}
+
+@TECHREPORT{calfits,
+  author = {{Ali}, Z. and  {Hazelton}, B. and {Beardsley}, A. and {La Planete}, P. and {Kunicki}, T. and {the pyuvdata team}},
+  title = "{Memo: UVCal FITS Format}",
+  year = 2017,
+  month = jul,
+  file = {https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/main/docs/references/calfits_memo.pdf},
+}
+
+@TECHREPORT{beamfits,
+  author = {{Hazelton}, B. and {the pyuvdata team}},
+  title = "{Memo: UVH5 file format}",
```

Comment:
I think this title is wrong.

Reply:
Whoops -- fixed in d9db6d2e9d65aabd9647ccf68a56013bccc209ab